docs/preview/connected-registry/quickstart-send-connected-registry-events-to-event-grid.md

```diff
@@ -0,0 +1,175 @@
+  - When you're prompted, install Azure CLI extensions on first use. For more information about extensions, see [Use extensions with Azure CLI](https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview).
+  - Run [az version](https://docs.microsoft.com/en-us/cli/azure/reference-index?#az_version) to find the version and dependent libraries that are installed. To upgrade to the latest version, run [az upgrade](https://docs.microsoft.com/en-us/cli/azure/reference-index?#az_upgrade).
+- The Azure CLI commands in this article are formatted for the Bash shell. If you're using a different shell like PowerShell or Command Prompt, you may need to adjust line continuation characters or variable assignment lines accordingly. This article uses variables to minimize the amount of command editing required.
+
+## Before you begin
+
+Make sure that you have created the connected registry resource in Azure as described in the [Create connected registry using the CLI](quickstart-connected-registry-cli.md) quickstart guide and have a connected registry deployed on your premises as described in [Quickstart: Deploy a connected registry to an IoT Edge device](quickstart-deploy-connected-registry-iot-edge-cli.md) or [Quickstart: Deploy a connected registry to Kubernetes cluster](quickstart-deploy-connected-registry-kubernetes.md). 
+
+Also, make sure you understand [Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/) and have registered to use it. You should also have an event end point ready where you would like to send your events. You can check the quickstart to configure Azure Event Grid for Azure Container Registry events as shown in [QuickStart: Container registry events on Event Grid][quickstart-eventgrid-container-registry].
+
+In this tutorial, we configure the connected registry to send artifact events like push and delete to Azure Event Grid in a two step process.
+
+1. Configure connected registry to generate events on patterns of interest.
+2. Configure Azure Event Grid to subscribe and filter for events generated by connected registry.
+
+## Notification Events Generation on a connected registry
+
+Use the following CLI command to configure the connected registry to generate events for all artifacts
+
+```azurecli
+az acr update -n $ACR_NAME
+  --name $ACR_CONNECTED_REGISTRY_NAME \
+  --add-notifications *:*
+```
+
+If you are interested in generating events only for specific artifact patterns or specific actions, you can specify the patterns in the form shown below
+
+```
+az acr update -n $ACR_NAME \
+  --name $ACR_CONNECTED_REGISTRY_NAME \
+  --add-notifications [PATTERN1 PATTERN2 ...]
+```
+
+where each pattern is defined as
+
+  ```
+imageName ":" action
```

Comment:
please update to `artifactName` since we support event notifications for artifact type

Reply:
ditto for the rest of the article